Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -152,9 +152,9 @@ static SessionResult webKitMediaOpenCDMDecryptorResetSessionFromKeyIdIfNeeded(We
 
     LockHolder locker(priv->m_mutex);
     RefPtr<WebCore::CDMInstance> cdmInstance = webKitMediaCommonEncryptionDecryptCDMInstance(self);
-    ASSERT(cdmInstance && is<WebCore::CDMInstanceOpenCDM>(*cdmInstance));
 
-    if(!cdmInstance){
+    if( !cdmInstance || !is<WebCore::CDMInstanceOpenCDM>(*cdmInstance) )
+        ASSERT_NOT_REACHED();
```

Comment:
We should ASSERT always after gstreamer logging, that way the log would be the last thing we see before the crash in debug mode.